src/site/shared/containers/Header/Right/index.tsx

```diff
@@ -1,24 +1,35 @@
 import {CircuitInfoHelpers} from "shared/utils/CircuitInfoHelpers";
+import {HeaderPopups} from "shared/state/Header/state";
 
 import {TutorialDropdown} from "./TutorialDropdown";
 import {OpenFileButton} from "./OpenFileButton";
 import {DownloadMenuDropdown} from "./DownloadMenuDropdown";
+import {UtilitiesDropdown} from "./UtilitiesDropdown";
 import {SignInOutButtons} from "./SignInOutButtons";
 import {SettingsMenu} from "./SettingsMenu";
 
 import "./index.scss";
 
+type Utility = {
+    popupName: HeaderPopups;
+    img: string;
+    text: string;
+}
 
 type Props = {
     helpers: CircuitInfoHelpers;
+    extraUtilities: Utility[];
```

Comment:
You can export the `Utility` type from `UtilitiesDropdown` and reference it from that